test.py

```diff
@@ -0,0 +1,3 @@
+
+def add1(x):
+    return x + 1
```

Comment:
We need a add2 function instead.
```suggestion
def add2(x):
    return x + 2
```